src/asammdf/blocks/mdf_v4.py

```diff
@@ -6593,7 +6590,7 @@ def get(
             if name is None:
                 name = channel.name
 
-            unit = conversion and conversion.unit or channel.unit
+            unit = conversion.unit if conversion else channel.unit
```

Comment:
This is not right: if the conversion unit is empty then we need to check the channel unit as well

Reply:
True. I just found out, that the preview rule, which flagged this, will be [removed soon](https://github.com/astral-sh/ruff/issues/9007)

Reply:
Fixed